service/Service/ServiceConfiguration.cs

```diff
@@ -162,6 +162,10 @@ private void ConfigureStorageDependency(IKernelMemoryBuilder builder)
                                                                 ?? this.GetServiceConfig<AzureBlobsConfig>("AzureBlob"));
                 break;
 
+            case string x when x.Equals("S3", StringComparison.OrdinalIgnoreCase):
```

Comment:
This probably also needs to be changed to AWSS3 ?

And also probably some settings in `tools/interactiveSetup/` can be changed to add AWSS3 support